vcd/resource_vcd_catalog_media_test.go

```diff
@@ -99,12 +125,12 @@ func testAccCheckVcdCatalogMediaExists(mediaName string, catalogItem *govcd.Cata
 			return fmt.Errorf("catalog %s does not exist (%s)", testSuiteCatalogName, err)
 		}
 
-		newCatalogItem, err := catalog.GetCatalogItemByName(catalogMediaRs.Primary.Attributes["name"], false)
+		newMedia, err := catalog.GetMediaByName(catalogMediaRs.Primary.Attributes["name"], false)
```

Comment:
"new" and "old" are the evil twins of naming variables. Please use precise word which describes what this is, "mediaRecord" maybe (IIRC the govcd PR right)?

Reply:
changed